app/models/upvs/message_draft.rb

```diff
@@ -0,0 +1,44 @@
+# == Schema Information
+#
+# Table name: messages
+#
+#  id                 :bigint           not null, primary key
+#  collapsed          :boolean          default(FALSE), not null
+#  delivered_at       :datetime         not null
+#  html_visualization :text
+#  metadata           :json
+#  outbox             :boolean          default(FALSE), not null
+#  read               :boolean          default(FALSE), not null
+#  recipient_name     :string
+#  replyable          :boolean          default(TRUE), not null
+#  sender_name        :string
+#  title              :string
+#  type               :string
+#  uuid               :uuid             not null
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  author_id          :bigint
+#  import_id          :bigint
+#  message_thread_id  :bigint           not null
+#
+class Upvs::MessageDraft < MessageDraft
```

Comment:
Uz viacrkat sme natrafili na to, ze by sme potrebovali pridat `Upvs::MessageDraft` triedu a teraz prisla ta chvila podla mna.